src/paint/supports.c

```diff
@@ -514,7 +599,7 @@ bool metal_a_supports_paint_setup(int supportType, int segment, int special, int
 		sint8 xOffset = loc_97AF20[segment].x;
 		sint8 yOffset = loc_97AF20[segment].y;
 
-		uint32 image_id = RCT2_ADDRESS(0x0097B15C + 2, uint16)[supportType * 2];
+		uint32 image_id = _97B190[supportType][1];
```

Comment:
@duncanspumpkin Can you validate that this change is correct? _97B190 is an array which differs slightly from _97B15C